workflow/pipeline/photometry.py

```diff
@@ -53,79 +53,262 @@ class EmissionColor(dj.Lookup):
 class FiberPhotometry(dj.Imported):
     definition = """
     -> session.Session
-    fiber_id            : tinyint unsigned    
     ---
-    sample_rate         : float     # (in Hz) 
-    timestamps          : longblob  # (in seconds) photometry timestamps, already synced to the master clock
     -> [nullable] LightSource
-    time_offset         : float     # (in second) time offset to synchronize the photometry traces to the master clock
-    notes=''            : varchar(1000)  
+    raw_sample_rate         : float         # sample rate of the raw data (in Hz) 
+    beh_synch_signal=null   : longblob      # signal for behavioral synchronization from raw data
     """
 
-    class Trace(dj.Part):
-        definition = """ # preprocessed photometry traces
+    class Fiber(dj.Part):
+        definition = """ 
         -> master
-        trace_name       : varchar(8)  # (e.g., raw, detrend, z)
+        fiber_id            : tinyint unsigned
+        -> reference.Hemisphere
```

Comment:
Does this needs to be on primary key? I think not

Reply:
No, it doesn't. It's just I'd like it to be inherited to downstream tables otherwise users will need to do join every time to get the information.